cli/internal/terraform/terraform/aws/variables.tf

```diff
@@ -1,37 +1,37 @@
 variable "name" {
-  type = string
+  type        = string
   description = "Name of your Constellation"
 }
 
-variable "worker_nodes_iam_instance_profile" {
+variable "iam_instance_profile_worker_nodes" {
   type        = string
   description = "Name of the IAM instance profile for worker nodes"
 }
 
-variable "control_plane_iam_instance_profile" {
+variable "iam_instance_profile_control_plane" {
   type        = string
   description = "Name of the IAM instance profile for control plane nodes"
 }
 
 variable "instance_type" {
   type        = string
   description = "Instance type for worker nodes"
-  default     = "t2.micro"
+  default     = "t3.micro"
 }
 
-variable "disk_size" {
+variable "state_disk_size" {
```

Comment:
This is currently unused. Is the current configured disk size (the local variable) the OS disk and we also need to attach more storage?

Reply:
Good catch. From how I interpret the docs, the AMI can define both the OS disk (root device) but also additional EBS storage devices.

For the first, if we provide an AMI we should not need any additional configuration with a read-only disk, I suppose. For the latter, I guess we can add a storage device configurable by the user using the variable and also storage type.

 I'll add this.